CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#119](https://github.com/rubocop-hq/rubocop-minitest/pull/119): Fix a bug in `bump_minor_version` in `Rakefile` that was preventing the `new_cop` task from completing successfully. ([@cstyles][])
```

Comment:
Thank you for the patch!
CHANGELOG.md does not need to be edited as it is not a change that directly affects RuboCop Minitest users. Can you remove the entry in CHANGELOG.md and squash your commits into one?

Reply:
Done!